arc_utils/produce_notification.py

```diff
@@ -0,0 +1,23 @@
+# producer.py
+
+import json
+
+from kafka import KafkaProducer
+
+
+class Notification:
+    def __init__(self):
+        self.topic = "arc-rte-notifications"
+        broker_url = ["arc-kafka.pt-staging.com:9092"]
```

Comment:
For `broker_url`, take in reference the `producer.py` file to avoid using hardcoded broker URL.